superset-frontend/src/dashboard/components/nativeFilters/FilterBar.tsx

```diff
@@ -183,6 +184,12 @@ const StyledCaretIcon = styled(Icon)`
   margin-top: ${({ theme }) => -theme.gridUnit}px;
 `;
 
+const StyledLoadingBox = styled.div`
+  position: relative;
+  height: 32px;
```

Comment:
Is there a reason not to make this `8 * gridUnit`?

Reply:
Actually no, my bad. I think when writing this I had in mind that we use gridUnit primarily for paddings and margins, but here the height is low and it's actually better to use gridUnit. I will change it, thanks!